sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -292,6 +301,7 @@ def __init__(self, client=None, temp_dataset_id=None):
         http=get_new_http(),
         credentials=auth.get_service_credentials(),
         response_encoding='utf8')
+    self.gcp_bq_client = client or gcp_bigquery.Client()
```

Comment:
Is this a backward incompatible change, because users might pass a different type of a client that does not support `insert_rows_json` ?